src/clients/api/mutations/vote/useCastVoteWithReason.ts

```diff
@@ -22,7 +23,28 @@ const useCastVoteWithReason = (
         fromAccountAddress,
         ...params,
       }),
-    options,
+    {
+      ...options,
+      onSuccess: (...onSuccessParams) => {
+        const { proposalId, voteType } = onSuccessParams[1];
+
+        // Invalidate query to fetch voters
```

Comment:
Same comment about invalidating these queries on an interval if the proposal is active